src/pages/LogView/LoadingPage/useResolveLogURLAndRenderingType.ts

```diff
@@ -189,17 +201,32 @@ export const useResolveLogURL = ({
           text: false,
         });
       downloadURL = rawLogURL;
+      if ((renderingTypeFromQuery || "") in LogRenderingTypes) {
+        renderingType = renderingTypeFromQuery as LogRenderingTypes;
+      } else if (!renderingTypeFromQuery) {
+        renderingType = LogRenderingTypes.Default;
+      } else {
+        renderingType = LogRenderingTypes.Default;
+        reportError(new Error("Encountered unsupported renderingType"), {
+          context: {
+            rawLogURL,
+            unsupportedRenderingType: renderingTypeFromQuery,
+          },
+        }).warning();
```

Comment:
This should probably be logged at `.severe()` since this indicates we encountered some sort of unsupported rendering type.